File: MTGSearch/src/main/java/com/dbottillo/mtgsearchfree/presenter/CardPresenter.java
package com.dbottillo.mtgsearchfree.presenter;

import com.dbottillo.mtgsearchfree.model.MTGCard;
import com.dbottillo.mtgsearchfree.view.CardView;

public interface CardPresenter {

    void loadOtherSideCard(MTGCard card);

    void init(CardView cardView);
}

File: MTGSearch/src/main/java/com/dbottillo/mtgsearchfree/view/views/CardPresenter.kt
package com.dbottillo.mtgsearchfree.view.views

import com.dbottillo.mtgsearchfree.model.MTGCard
import com.dbottillo.mtgsearchfree.view.CardView

interface CardPresenter {
    fun loadOtherSideCard(card: MTGCard)
    fun init(cardView: CardView)
}
